### ISSUE TITLE
Breitbart should not be on the list

### ISSUE THREAD

**Issue by ohadschn** [marktron/fakenews#37]
You might disagree with their political views, but there are tens if not hundreds of millions who don't consider them fake news: 
![image](https://user-images.githubusercontent.com/8555641/97114230-01090180-16f8-11eb-87af-0b32660d5406.png)

That's >10% more engagements than NY Times, and a case could just as easily be made against CNN for "fake news" (stories that turned out to be completely false). Is it clearly biased and ideologically motivated (as is thy NY Times)? Yes. Is it fake new? no. 

Bottom line is - right wingers will say CNN is fake, left wingers will say Breitbart is fake, so you have to go with bipartisan consensus. As the numbers above demonstrate, Breitbart is clearly outside this consensus.

**Comment by ohadschn** [comment]
Thank you - closing this issue without any debate or explanation actually tells me everything I needed to know about this list.
